dev/flake.nix

```diff
@@ -32,6 +33,32 @@
             nixpkgs-fmt.enable = true;
           };
         };
+
+        # TODO: Move this under `CIApps` once `omnix` supports running them automatically
```

Comment:
What is `CIApps`?

Reply:
Custom flake-schema that can be detected by omnix to run everything under it as custom app steps.